Gerik/Crypters/Cezar/Caesar.java

```diff
@@ -0,0 +1,102 @@
+public class Caesar {
+
+    public static void Info() //show helpful info
+    {
+        System.out.print("Usage: $ Java Caesar [OPTION] <Text> <Step>\n\nProgram for encrypting and decrypting text by using a Caesar method\n\n\t-h\tHelp\n\t-c\tUse for encrypting text\n\t-d\tUse for decrypting text\n\nStep must be a integer number\n\nExamples:\n$ java Caesar -c TryMeBitch 4\n$ java Caesar -d XizJuTpFbtz@ 1\n");
+    }
+    public static class CycleValue
+    {
+        protected int currentValue = 0;
+        protected int border;
+        CycleValue(int _border, int value)
+        {
+            currentValue = value;
+            border = _border;
```

Comment:
public CycleValue

also
this.currentValue = currentValue;
this.border = border;